src/main/java/com/google/sps/servlets/CheckPermissionsServlet.java

```diff
@@ -80,12 +80,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         cloudResourceManagerService.projects().testIamPermissions(projectId, permissionsRequestBody);
 
       TestIamPermissionsResponse permissionsResponse = testingPermissionsResponse.execute();
+      Boolean arePermissionsCorrect = false;
 
-      System.out.println(permissionsResponse);
+      if (permissionsResponse.getPermissions().size() == requiredPermissions.size()) {
+        arePermissionsCorrect = true;
+      }
 
```

Comment:
More readable as:
```
arePermissionsCorrect = permissionsResponse.getPermissions().size() == requiredPermissions.size()
```

In general, when you have something like 
```
x = false
if (<boolean expression>) {
  x = true
}
```
you can rewrite it as
```
x = <boolean expression>
```

Reply:
I hadn't considered doing it that way but it is definitely more readable - thanks!